covid19-etl/utils/metadata_helper.py

```diff
@@ -101,6 +101,26 @@ def get_str_latest_submitted_date_idph(self):
         loc = query_res["data"]["location"][0]
         return loc["date"]
 
+    def get_str_latest_submitted_date_idph_group_demo(self):
+        print("Getting the latest summary_clinical date from Guppy...")
+        query_string = """query ($filter: JSON) {
+            group_demographics (
```

Comment:
you could have added a parameter to `get_str_latest_submitted_date_idph` instead of duplicating the code